src/orion/benchmark/task/quadratics_test.py

```diff
@@ -0,0 +1,140 @@
+from .quadratics import QuadraticsTask
+from .task import Task
+from orion.benchmark.task.utils import distance, similarity
+from typing import ClassVar, Type
+import numpy as np
+import pytest
+
+
+class TestQuadraticsTask:
+    task: ClassVar[Type[Task]] = QuadraticsTask
+
+    def test_seeding(self):
+        seed = 123
+
+        task_a = self.task(seed=seed)
+        task_b = self.task(seed=seed)
+        assert task_a.a2 == task_b.a2
+        assert task_a.a1 == task_b.a1
+        assert task_a.a0 == task_b.a0
+        assert task_a == task_b
+
+        task_c = self.task(seed=seed + 1)
+        assert task_b.a2 != task_c.a2
+        assert task_b.a1 != task_c.a1
+        assert task_b.a0 != task_c.a0
+        assert task_b != task_c
+
+    @pytest.mark.parametrize(
+        "task_a_kwargs, task_b_kwargs, expected_distance",
+        [
+            (dict(a2=0, a1=1.2, a0=0), dict(a2=0, a1=1.2, a0=0), 0.0),
+            (dict(a2=0, a1=1.2, a0=0), dict(a2=0, a1=0.8, a0=0), 0.4),
+            (dict(a2=1, a1=1.2, a0=0), dict(a2=1, a1=0.8, a0=0), 0.4),
+            (dict(a2=1, a1=1.2, a0=2), dict(a2=1, a1=0.8, a0=2), 0.4),
+            (dict(a2=1, a1=0, a0=0), dict(a2=2, a1=0, a0=0), 3.0),
+            (dict(a2=1, a1=0, a0=0), dict(a2=3, a1=0, a0=0), 8.0),
+            (dict(a2=1, a1=0, a0=0), dict(a2=3, a1=1, a0=0), 9.0),
+            (dict(a2=1, a1=0, a0=0), dict(a2=3, a1=-1, a0=0), 9.0),
+        ],
+    )
+    def test_distances_between_tasks(
+        self, task_a_kwargs: dict, task_b_kwargs: dict, expected_distance: float
+    ):
+        task_a = self.task(**task_a_kwargs)
+        task_b = self.task(**task_b_kwargs)
+        actual_distance = distance(task_a, task_b)
+        assert np.isclose(actual_distance, expected_distance)
+
+    @pytest.mark.parametrize(
+        "task_a_kwargs, task_b_kwargs, expected_similarity",
+        [
+            (dict(a2=0, a1=1.2, a0=0), dict(a2=0, a1=1.2, a0=0), 1.0),
+            # TODO: The resulting similarity value for these is a bit arbitrary:
+            # (dict(a2=0.1, a1=1.0, a0=0.1), dict(a2=0.1, a1=2.0, a0=0.1), 0.25),
+            # (dict(a2=0.1, a1=1.0, a0=0.1), dict(a2=0.1, a1=2.0, a0=0.1), 0.25),
+            # These are the most dis-similar tasks imaginable.
+            (dict(a2=0.1, a1=0.1, a0=0.1), dict(a2=10.0, a1=10.0, a0=10.0), 0.0),
+            (dict(a2=10.0, a1=10.0, a0=10.0), dict(a2=0.1, a1=0.1, a0=0.1), 0.0),
+        ],
+    )
+    def test_similarity_between_tasks(
+        self, task_a_kwargs: dict, task_b_kwargs: dict, expected_similarity: float
+    ):
+        task_a = self.task(**task_a_kwargs)
+        task_b = self.task(**task_b_kwargs)
+        actual_similarity = similarity(task_a, task_b)
+        assert np.isclose(actual_similarity, expected_similarity)
+
+    MAX_DIST = distance(task.task_low(), task.task_high())
+
+    @pytest.mark.parametrize(
```

Comment:
If this takes to much time to fix, the feature `get_similar_task` could be added in a latter PR.